fastlane/swift/SocketClient.swift

```diff
@@ -82,12 +89,14 @@ class SocketClient: NSObject {
         }
         
         self.dispatchGroup.enter()
-        self.streamQueue.async {
+        print("Enter \(#function) \(#line)")
+        self.readQueue.sync {
             self.inputStream.open()
         }
         
         self.dispatchGroup.enter()
-        self.streamQueue.async {
+        print("Enter \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
        verbose(message: "Enter \(#function) \(#line)")
```